task_object.py

```diff
@@ -0,0 +1,35 @@
+import pickle
+import hashlib
+
+
+class Task:
+    def __init__(self, summary, due_date, progress, user_id):
+        self.summary = summary
+        self.due_date = due_date
+        self.progress = progress
+        self.user_id = user_id
+        self.id = hashlib.sha3_256(str(self.__dict__).encode('utf-8')).hexdigest()[:8]
```

Comment:
this line may cause a problem, what if we have two tasks with same `summary` but different `due date` or `user_id`?